odoo/addons/base/migrations/12.0.1.3/noupdate_changes.xml

```diff
@@ -171,4 +171,7 @@ System</span>]]></field>
   <record id="dm" model="res.country">
     <field eval="1767" name="phone_code"/>
   </record>
+  <record id="mr" model="res.country">
+    <field name="currency_id" ref="MRU"/>
```

Comment:
Isn't there any constraint that prevents this?

Reply:
I don't think so :S

Reply:
Yeah, and Travis would complain about it, so good to go.